src/GitVersion.Core/VersionCalculation/VersionSearchStrategies/TaggedCommitVersionStrategy.cs

```diff
@@ -10,14 +11,15 @@ namespace GitVersion.VersionCalculation;
 /// Increments if the tag is not the current commit.
 /// </summary>
 internal sealed class TaggedCommitVersionStrategy(
-    Lazy<GitVersionContext> contextLazy,
+    Lazy<GitVersionContext> contextLazy, ILog log,
```

Comment:
When parameters are already newline-separated, please have every parameter on its own line.

```suggestion
    Lazy<GitVersionContext> contextLazy,
    ILog log,
```